Carsus_atomic_data/Custom_atomic_data.ipynb

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
<p>Weird command and then period at the end of this line. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>